Lib/hashlib.py

```diff
@@ -257,6 +258,33 @@ def prf(msg, inner=inner, outer=outer):
         logging.exception('code for hash %s was not found.', __func_name)
 
 
+def _digestmod_to_name(digestmod, from_instance=False):
+    """Convert a string, callable or digest module to a digest name
```

Comment:
append `or None.`

Reply:
``_digestmod_to_name()`` is pining for the fjords.